nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/resources/test-invite-contrib.xml

```diff
@@ -12,6 +12,9 @@
   <extension target="org.nuxeo.ecm.user.invite.UserInvitationService"
              point="configuration">
 
+    <configuration remove="true">
+    </configuration>
+
```

Comment:
hi, yes it is that.

without it, i would get the  warning
```Trying to register an existing userRegistration configuration without removing or merging it,````
and eventually tests would fail as the test contribution would get mixed with the default one (from `user-registration-contrib.xml`)